tests/test_config_registry_and_test_execution.py

```diff
@@ -61,8 +64,14 @@ def test_config_registry_and_test_execution_with_pattern(tmp_test, capsys):
     assert "Ran 0 tests" in output.err
 
 
-@pytest.mark.parametrize("tmp_test", [("aux1", "aux2", False)], indirect=True)
-def test_config_registry_and_test_execution_collect_error(tmp_test, capsys, mocker):
+@pytest.mark.parametrize(
+    "tmp_test",
+    [("collector_error_aux1", "collector_error_aux_2", False)],
+    indirect=True,
+)
+def test_config_registry_and_test_execution_collect_error(
```

Comment:
```suggestion
def test_config_registry_test_collection_error(
```
Just to split this test from the one you added